lib/orator/index.js

```diff
@@ -327,18 +327,29 @@ class Orator {
           avatarURL
         } = response
 
-        const hook = await findOrCreateManagedWebhook(
-          msg.channel,
-          bot.user.id,
-          this.webhookName
-        )
+        try {
+          const hook = await findOrCreateManagedWebhook(
+            msg.channel,
+            bot.user.id,
+            this.webhookName
+          )
 
-        newMessage = await bot.executeWebhook(hook.id, hook.token, {
-          ...content,
-          username,
-          avatarURL,
-          wait: true
-        })
+          newMessage = await bot.executeWebhook(hook.id, hook.token, {
+            ...content,
+            username,
+            avatarURL,
+            wait: true
+          })
+        } catch (e) {
+          switch (e.message) {
+            case 'Missing Permissions':
+              this.createMessage(bot.user, msg.channel,
```

Comment:
instead of creating a message like this we should probably use the message send handlers that we use in the else handler below this.

that way we will still have `newMessage` for the rest of the flow.